### ISSUE TITLE
About the problem of unsuccessful connection after docker is started

### ISSUE THREAD

**Issue by MrDengML** [sdlhappylin/V-Rising-Server-Docker#4]
According to the original method, the corresponding ip address needs to be added to the [start_server_example.bat] file, then how do you deal with this method? I git down and docker is also started, that is, enter the ip directly in the game connected, but can't connect.

**Comment by sdlhappylin** [comment]
Do not use the script [start_server_example.bat] ! ip should be modified in Settings/ServerHostSettings.json. You don't need to change ip ,the problem maybe you foget to expose your port or set 777 permission for server-data

**Comment by MrDengML** [comment]
I found a solution. In my case, the game server is deployed on a cloud server, so I need to add [-address "ip"] at the end of the content of the [start_server_example.bat] file to find the server in the game and connect. I The final solution is to enter the docker container and modify the files I want to modify internally. And since I couldn't use vi or vim command, I used [docker cp sourceFile targetFolder ] to replace it. I finally successfully connected to the server. I don't know if I installed it wrongly or for other reasons, but I finally implemented it like this. Finally thank you for the way
